src/Icon/index.jsx

```diff
@@ -70,28 +70,35 @@ function Icon({
 }
 
 Icon.propTypes = {
-  // eslint-disable-next-line max-len
-  /** An icon component to render. Example import of a Paragon icon component: `import { Check } from '@edx/paragon/dist/icon';` */
+  /**
+   * An icon component to render.
+   * Example import of a Paragon icon component: `import { Check } from '@edx/paragon/dist/icon';`
```

Comment:
Currently, we import icons with `import { Check } from '@edx/paragon/icons';`, this is probably a very old docstring, so we should update the import as well

Reply:
Yeah, thanks